### DISCUSSION TITLE
Community Edition - IP Protection

### DISCUSSION THREAD

**Discussion by ShaileshT-dot** [pimcore/pimcore#16203] [Q&A / Support]
Hi all, As per this link [https://pimcore.com/editions](url), can someone please give details as to what all are covered/not covered under the IP Production for community edition?
Suppose that an organization has installed community edition and have built custom components, custom REST APIs, are they bound to share the developed components in the community? 
please share details.

**Comment by kingjia90** [comment]
I don't know but the Pimcore sales team would be better at replying this, you can directly contact them with the form below the official website.
